audio/libfreeaptx/libfreeaptx.SlackBuild

```diff
@@ -92,12 +92,11 @@ rm -f $PKG/{,usr/}lib${LIBDIRSUFFIX}/*.la
 find $PKG -print0 | xargs -0 file | grep -e "executable" -e "shared object" | grep ELF \
   | cut -f 1 -d : | xargs strip --strip-unneeded 2> /dev/null || true
 
-mkdir -p $PKG/usr/doc/$PRGNAM-$VERSION
```

Comment:
I don't think should not arbitrarily deviate from the template.

Things coming from `$CWD` should typically be installed with `cat` or `install` instead of copying them. The items in `$CWD` have of course not had their owner, group, and permissions normalised as the files from source tarball have been.

You can change `cp COPYING README $PKG/usr/doc/$PRGNAM-$VERSION` to `cp -a COPYING README $PKG/usr/doc/$PRGNAM-$VERSION` to preserve these attributes on those files though.